spring-integration-file/src/test/java/org/springframework/integration/file/locking/NioFileLockerTests.java

```diff
@@ -76,4 +77,20 @@ public void fileNotListedWhenLockedByOtherFilter() throws IOException {
 		filter1.unlock(testFile);
 	}
 
+	@Test
+	public void fileLockedWhenNotAlreadyLockedAndExists() throws IOException {
+		NioFileLocker locker = new NioFileLocker();
+		File testFile = new File(workdir, "test2");
+		testFile.createNewFile();
+		assertThat(locker.lock(testFile)).isTrue();
+		locker.unlock(testFile);
+	}
+
+	@Test
+	public void fileNotLockedWhenNotExists() throws IOException {
+		NioFileLocker locker = new NioFileLocker();
+		File testFile = new File(workdir, "test3");
+		assertThat(locker.lock(testFile)).isFalse();
+		assertThat(testFile.exists()).isFalse();
+	}
```

Comment:
Every member in the class should be wrapped with blank lines, even the last method before the end of class.